src/main/java/com/example/bak/global/exception/ErrorCode.java

```diff
@@ -14,12 +14,19 @@
 @Getter
 @RequiredArgsConstructor
 public enum ErrorCode {
+    // Auth
+    INCORRECT_PASSWORD("AU001", HttpStatus.UNAUTHORIZED, "비밀번호가 일치하지 않습니다"),
+    INVALID_TOKEN("AU002", HttpStatus.UNAUTHORIZED, "토큰이 유효하지 않습니다"),
+    NO_ATTRIBUTES("AU003", HttpStatus.BAD_REQUEST, "토큰 검증에 필요한 정보가 없습니다."),
+    NO_TOKEN("AU004", HttpStatus.UNAUTHORIZED, "토큰이 없습니다."),
+    NO_AUTHORITY("AU005", HttpStatus.FORBIDDEN, "해당 정보에 접근할 수 있는 권한이 없습니다."),
     // Company
```

Comment:
```suggestion
    NO_AUTHORITY("AU005", HttpStatus.FORBIDDEN, "해당 정보에 접근할 수 있는 권한이 없습니다."),
    
    // Company
```

공백 한 칸 들어가면 구분되고 참 좋을 것 같아용